pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -63,7 +77,9 @@ attrsets.filterAttrs (attr: _: (builtins.hasAttr attr prev)) {
     oldAttrs: {
       propagatedBuildInputs = [final.setupCudaHook];
 
-      meta = (oldAttrs.meta or {}) // {
+      allowFHSReferences = true;
```

Comment:
seems like we're adding a new configuration option/flag... can we document this new flag somewhere, perhaps with an example or two of its intended usage?

Reply:
It's only used to disable this: https://github.com/NixOS/nixpkgs/blob/684ca4d96e6447cf6eb0ceb23128bb975cfd7a00/pkgs/development/cuda-modules/generic-builders/manifest.nix#L226-L232

I'm not sure about documenting things we might want to delete, but I guess I should elaborate that the reason we disable the check for nvcc is that the binaries themselves (nvcc and cicc) contain matches